src/containers/Group/Producers/Item.js

```diff
@@ -16,6 +17,7 @@ class Item extends Component {
 
     return (
       <div>
+        <Link to={`/groups/1/producers/${producer.id}`}>hola</Link>
```

Comment:
yep but it would be nice to relative links working since you'll need to pass the `group` to all the childrens just to build the url in the links.

Anyway, let's try to get it working first, since it doesn't work even with this absolute hardcoded path 
```
to={`/groups/1/producers/${producer.id}\`}
```